pallets/mining/eligibility/proxy/src/lib.rs

```diff
@@ -111,14 +211,71 @@ decl_storage! {
+            Option<MiningEligibilityProxyRewardRequest<
                 T::AccountId,
                 BalanceOf<T>,
-                // Vec<RewardeeData<T>>,
-                T::BlockNumber,
+                Vec<RewardeeData<T>>, // TODO - change to store MiningEligibilityProxyRewardeeIndex that is created to store then instead
+                <T as pallet_timestamp::Config>::Moment,
             >>;
+
+        // IGNORE - seems unnessary since can get from MiningEligibilityProxyRewardRequests
+        // pub MiningEligibilityProxyRewardees get(fn mining_eligibility_proxy_eligibility_rewardees): map hasher(opaque_blake2_256) T::MiningEligibilityProxyRewardeeIndex =>
+        //     Option<MiningEligibilityProxyRewardee<
+        //         u64,                        // reward_hash
+        //         T::AccountId,               // requestor_account_id
+        //         BalanceOf<T>,               // reward_amount
+        //         T::Moment,                  // timestamp
+        //         T::BlockNumber,             // block_start
+        //         T::BlockNumber,             // block_interval
+        //     >>;
+
+        /// Stores reward_requests for given rewardee
+        ///
+        /// requestor_acct_id > (reward_hash, total_amt, rewardee_count, member_kind, date)
+        /// where member_kind is either supernode_center 1 or supernode 2 or individual 3
+        pub MiningEligibilityProxyRewardRequestors get(fn reward_requestors):
+            map hasher(opaque_blake2_256) T::AccountId =>
+                Option<Vec<RewardRequestorData<
+                    <T as Config>::MiningEligibilityProxyIndex,
+                    BalanceOf<T>,
+                    u64,
+                    u32,
+                    <T as pallet_timestamp::Config>::Moment,
+                >>>;
+
+        /// Stores reward_transfers for given rewardee
+        /// IMPORTANT NOTE: REQUESTOR MAY DIFFER FROM REQUESTEE
+        /// HENCE DIFFERENT STORAGE FROM `reward_requestors`
+        ///
+        /// rewardee_acct_id > (reward_hash, bool, total_amt, rewardee_count, member_kind, date)
+        pub MiningEligibilityProxyRewardTransfers get(fn reward_transfers):
+            map hasher(opaque_blake2_256) T::AccountId =>
+                Option<Vec<RewardTransferData<
+                    <T as Config>::MiningEligibilityProxyIndex,
+                    bool,
+                    BalanceOf<T>,
+                    u64,
+                    u32,
+                    <T as pallet_timestamp::Config>::Moment,
+                >>>;
+
+        /// Substrate-fixed, value starts at 0 (additive identity)
+        pub TotalRewardsPerDay get(fn total_rewards_daily):
+            map hasher(opaque_blake2_256) Date => Option<BalanceOf<T>>;
+
+        /// Stores accumulation of daily_rewards_sent on a given date
+        /// Note: Must store date/time value as the same for all rewards sent on same day
+        pub RewardsPerDay get(fn rewards_daily):
```

Comment:
a quick search in https://github.com/DataHighway-DHX/node/blob/1f4d8afa7b07fd3b8704aab353385b19096664ae/custom_types.json and I can't find `RewardDailyData` nor `RewardTransferData`. Please check the console log of the polkadot apps which should tell you all the missing types.